### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Amir-P**

### CODE REVIEW

File: packages/fleather/README.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
 
-[comment]: <> (![zefyr screenshot]&#40;https://github.com/memspace/zefyr/raw/master/packages/zefyr/zefyr.png&#41;)
+## Features
+* Inline attributes like **bold**, *italic*, ~~strikethrough~~ and etc.
+* Line attributes like direction, alignment, heading, number and bullet list and etc.
+* Block attributes like code, quote and etc.
+* Markdown-inspired semantics
+* Support for inline and block embeds
+* Support for markdown shortcuts
+* Using [Quill.js Delta](https://quilljs.com/docs/delta) as underlying data format by [Parchment](packages/parchment/README.md), Fleather is ready for collaborative editing using [OT](https://en.wikipedia.org/wiki/Operational_transformation) or [CRDT](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) (Not provided as built-in functionality yet)
```
Comment (cgestes) [inline packages/fleather/README.md:14]: any idea how to do CRDT? I could easily part with OT :)

Comment (Amir-P) [reply packages/fleather/README.md:14]: No idea! I've never done OT or CRDT. @cgestes

Comment (amantoux) [reply packages/fleather/README.md:14]: Yeah I think we can remove the CRDT part indeed
Never tried it and I'm not sure I intend to do so :)

Comment (cgestes) [reply packages/fleather/README.md:14]: I am not sure the delta format allows for CRDT. I know for sure OT works :)

File: packages/fleather/README.md [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
+    Expanded(
+      child: FleatherEditor(controller: controller),
+    ),
+    //or
+    FleatherField(controller: controller)
+  ],
+),
+```
+
+## Credits
```
Comment (amantoux) [inline packages/fleather/README.md:49]: I'm not in favor of a credits section generally speaking
unless you absolutely wish to keep it

File: packages/fleather/README.md [hunk 3]
```diff
@@ -1,7 +1,52 @@
...
+* Supports markdown shortcuts
+* Using [Quill.js Delta](https://quilljs.com/docs/delta) as underlying data format by [Parchment](packages/parchment/README.md), Fleather is ready for collaborative editing using [OT](https://en.wikipedia.org/wiki/Operational_transformation) (Not provided as a built-in functionality)
 
+## Get started
+Add Fleather to your dependencies.
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  fleather: ^1.2.1
```
Comment (amantoux) [inline packages/fleather/README.md:25]: `1.2.2` now

We should add it to the release checklist or find a way to automate it

**Comment by amantoux** [comment]
@Amir-P some minor comments

File: packages/fleather/README.md (hunk 2 above)
Comment (Amir-P) [reply packages/fleather/README.md:49]: So how we should give credit for something like banner? @amantoux

Comment (amantoux) [reply packages/fleather/README.md:49]: Ok keep it

File: packages/fleather/README.md (hunk 3 above)
Comment (Amir-P) [reply packages/fleather/README.md]: Done. I'll search for a way to update it automatically. @amantoux

**Pull request merged**